src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -467,6 +468,16 @@ public WorkspaceViewModel(WorkspaceModel model, DynamoViewModel dynamoViewModel)
             InCanvasSearchViewModel = new SearchViewModel(DynamoViewModel);
             InCanvasSearchViewModel.Visible = true;
         }
+        /// <summary>
+        /// This event is triggred from Workspace Model. Used in instrumentation
```

Comment:
Can we change this summary to indicate this is a handler for an event rather than an event, and what the function's actual purpose is... why is it used for instrumentation.

something like - 
` this function handles the workspaceModels request to populate a json object with view data. It's used to construct a full workspace for instrumentation.
`